main.py

```diff
@@ -241,7 +230,7 @@ def escape_path(path):
 		# Build qemu launch cmd
 		cmd = [xqemu_path,
 		       '-cpu','pentium3',
-		       '-machine','xbox%(accelerator_arg)s,bootrom=%(mcpx_path_arg)s%(short_anim_arg)s' % locals(),
+		       '-machine','xbox,accel=kvm:hax:whpx:tcg%(kvm_irqchip_arg)s,bootrom=%(mcpx_path_arg)s%(short_anim_arg)s' % locals(),
```

Comment:
The accelerator should still respect the UI checkbox.

There's many issues with HW accelerators + we don't want dead UI components.